docs-ref-conceptual/interactive-azure-cli.md

```diff
@@ -78,19 +78,31 @@ az>>
 
 ## Query
 
-You can execute a JMESPath query on the results of the last command that you executed.
+You can execute a JMESPath query on the results of the last command that you executed by using `??`followed by a JMESPath query.
 For example, after you create a VM, you can make sure it has fully provisioned.
 
 ```azurecli
 az>> vm create --name myVM --resource-group myRG --image UbuntuLTS --no-wait -o json
-az>> ? [*].provisioningState
+az>> "?? [*].provisioningState"
```

Comment:
This command does not work for `vm create --no-wait` since there is no output. It will return `null` as a result, rather than the expected output below. Change the command to something that would produce output, such as `group create`, which will have a `.provisioningState` result.

Reply:
Resolved using `group create` as suggested but using `.id` as it seems there is no `provisioningState` on a group.